src/integrations/watchers/indexable-homeurl-watcher.php

```diff
@@ -79,35 +79,12 @@ public function register_hooks() {
 	 * @return void
 	 */
 	public function reset_permalinks() {
-		$post_types = $this->get_post_types();
```

Comment:
To my understanding, `get_post_types()` is now unused. Shouldn't this method (see below in this file) be entirely removed?